README.md

```diff
@@ -48,19 +49,31 @@ $ lb app loopback-example-access-control
 ... # follow the prompts
 $ cd loopback-example-access-control
 ```
+### Add the datasource
+- Name: `mycloudant`
+  - datasource: `cloudant`
+  - url: `your_cloudant_url`
+  - database: `your_cloudant_database`
+  - username: `your_cloudant_username`
```

Comment:
True, while some user doesn't have url so they provide usr&pwd, I will link to cloudant#config here so they won't get confused: https://github.com/strongloop/loopback-connector-cloudant#configuration